packages/daimo-userop/src/index.ts

```diff
@@ -60,30 +60,24 @@ export class DaimoAccount {
   public static async init(
     deployedAddress: Address,
     signer: SigningCallback,
+    rpcUrl: string,
```

Comment:
what is `userop` still using the rpc for?

- we get gas cost + paymaster info from Daimo API
- `sendUserOperation` goes thru the bundler API